google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -152,3 +152,4 @@ clouddebug.background.listener.general.error.message=There was an unexpected err
 clouddebug.debug.targets.error=Error accessing debug targets: {0}
 clouddebug.debug.targets.accessdenied=Access denied. You may not have the necessary permissions to run Cloud Debugger with this project.
 
+settings.error.closing.file=Error closing settings file {0}.\nThere was {1}
```

Comment:
This message is a bit weird.  Maybe just replace 'There was ..' with 'Error details:.. '

Typically for these kind of errors we probably just want to log error rather than show a dialog, but I don't want to waste time on this class since it's not actually in use.